app/services/BulkRequestCreationService.scala

```diff
@@ -102,6 +102,7 @@ trait BulkRequestCreationService extends BulkEntityProcessor[BulkCalculationRequ
 
     calculationRequestLine.calctype match {
       case Some(0) => calculationRequestLine.copy(revaluationDate = None, revaluationRate = None)
+      case Some(2) | Some(4) => calculationRequestLine.copy(revaluationDate = None)
```

Comment:
@RaminEsfandiari I raised that query and it was just the date that needs emptied, as they could have a rate without a date.